server/server.js

```diff
@@ -1,43 +1,58 @@
 const _ = require('lodash')
 const http = require('http')
 const express = require('express')
-const bodyParser = require('body-parser')
-const {graphqlExpress, graphiqlExpress} = require('apollo-server-express')
+const { ApolloServer } = require('apollo-server-express')
 const cors = require('cors')
 const {log} = require('./lib/util/logger')
 const expressPino = require('express-pino-logger')({logger: log})
 const {runHealthChecks} = require('./health')
 const {getMetrics, responseLoggingMetric} = require('./metrics')
-const { subscribe, execute } = require('graphql')
-const { SubscriptionServer } = require('subscriptions-transport-ws')
 
 const schemaParser = require('./lib/schemaParser')
 const schemaListenerCreator = require('./lib/schemaListeners/schemaListenerCreator')
 
-module.exports = async ({graphQLConfig, graphiqlConfig, postgresConfig, schemaListenerConfig}, models, pubsub) => {
-  const {tracing} = graphQLConfig
-  let {schema, dataSources} = await buildSchema(models, pubsub)
-  await connectDataSources(dataSources)
-
-  const app = express()
+function newExpressApp (schema, httpServer, tracing, playgroundConfig) {
+  let app = express()
 
   // Wrap the Express server
-  const server = http.createServer(app)
-  let subscriptionServer = null // will be instantiated later
+  // const server = http.createServer(app)
+  // let subscriptionServer = null // will be instantiated later
 
   app.use(responseLoggingMetric)
 
   app.use('*', cors())
   app.use(expressPino)
 
-  app.use('/graphql', bodyParser.json(), function (req, res, next) {
-    const context = {request: req}
-    const graphql = graphqlExpress({schema, context, tracing})
-    return graphql(req, res, next)
+  let apolloServer = new ApolloServer({
+    schema,
+    context: async ({ req }) => {
+      return {
+        request: req
+      }
+    },
+    tracing,
+    playground: {
```

Comment:
Please update the Graphiql related part in README.

Playground URL is http://localhost:8000/graphql